SDK/AppCenterData/Microsoft.AppCenter.Data.Android.Bindings/Transforms/Metadata.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<metadata>
+    <attr path="/api/package[@name='com.microsoft.appcenter.data']/class[@name='Data']" name="managedName">AndroidData</attr>
```

Comment:
I'm not sure what all needs to be in here but it's certainly missing some of the new shared classes, need to investigate

Reply:
Although it doesn't look like we need references to the new static classes because these are for the conversion extensions?

Reply:
We should rename all public classes part of the documented/supported surface to Android*.
The goal is to avoid Intellisense to pick up the Java class name when resolving using statements that are missing, very old issue that we resolved by prefixing on other modules.